tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +335,67 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateAddPOsAsProjectOwner() {
+	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.standardUserCOProject = createProjectAsCO
+
+	log.Info("Additional Testcase4 - Validating if Project Owner can add another Project Owner ")
+	//Additional test4 validate if member with role Project Owner can add a project owner
+	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleProjectOwner)
+	require.NoError(rb.T(), errUserRole)
+	rb.additionalUserClient, err = rb.additionalUserClient.ReLogin()
+
+	additionalUser1, err := createUser(rb.client, standardUser)
+	additionalUserClient1, err := rb.client.AsUser(additionalUser1)
+	require.NoError(rb.T(), err)
+
+	errUserRole2 := users.AddProjectMember(rb.additionalUserClient, rb.standardUserCOProject, additionalUser1, roleProjectOwner)
+	require.NoError(rb.T(), errUserRole2)
+	additionalUserClient1, err = additionalUserClient1.ReLogin()
+
+	//Check user is associated with the project
+	userGetProject, err := projects.GetProjectList(additionalUserClient1, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
+	assert.Equal(rb.T(), 1, len(userGetProject.Data))
+
+	//Remove user from the project
+	errRemoveMember := users.RemoveProjectMember(rb.standardUserClient, additionalUser1)
+	require.NoError(rb.T(), errRemoveMember)
+
+	//Validated user is removed from the project
+	userProjectEmptyAfterRemoval, err := projects.GetProjectList(additionalUserClient1, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	assert.Equal(rb.T(), 0, len(userProjectEmptyAfterRemoval.Data))
+}
+
+func (rb *RBTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
+	//Create project as a cluster Owner
+	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	rb.standardUserCOProject = createProjectAsCO
+
+	log.Info("Additional Testcase5 - Validating if Manage Project Member cannot add Project Owner ")
+	//Additional test5 validate if member with role Manage Project Member cannot add a project owner
+	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleManageProjectMember)
+	require.NoError(rb.T(), errUserRole)
+	rb.additionalUserClient, err = rb.additionalUserClient.ReLogin()
+
+	additionalUser1, err := createUser(rb.client, standardUser)
+	additionalUserClient1, err := rb.client.AsUser(additionalUser1)
```

Comment:
after the additionalUser1 is changed we can rename additionalUserClient1 var too.